README.md

```diff
@@ -14,83 +14,260 @@ A curated list of anime streaming/downloading websites.
 * I am **NOT** promoting the use of free anime websites by any means.
 
 
-## Contents
+# Contents
+* [Anime](##Anime)
```

Comment:
Clicking on these links is not taking me to the correct section
```suggestion
* [Anime](#Anime)
```

This should work